### ISSUE TITLE
detecting wp themes & plugins

### ISSUE THREAD

**Issue by hbakhtiyor** [AliasIO/Wappalyzer#1474]
any possibility to add detecting wp themes & plugins algorithm to the code base?

**Comment by gadcam** [comment]
Your request seems to be out of scope, see #324.
Imho the detection of themes or plugins of a specific CMS, framework, .. should be done in a separate addon or as an addon of W. see https://addons.mozilla.org/en-US/firefox/addon/firefinder-for-firebug/ for Firebug.

**Comment by hbakhtiyor** [comment]
@gadcam any documentations for addon of wapplazer? couldn't find it

**Comment by gadcam** [comment]
It is not implemented yet! I don't really know how it works with Firebug.
We could imagine something injecting code to add a specific apps.json and the related icons.
I'm only a contributor, only @AliasIO could answer you if he want this type of feature to be added.

**Comment by gadcam** [comment]
It is not implemented yet! I don't really know how it works with Firebug.
We could imagine something injecting code to add a specific apps.json and the related icons.
I'm only a contributor, only @AliasIO could answer you if he want this type of feature to be added.

**Comment by stefanos82** [comment]
Firebug got merged with Firefox's web developer tools. Beside this, what @gadcam has said it's true; what you are asking for is out of extension's scope.

Also, since I'm an experienced WP user myself, allow me to give you a small hint: you can customize how your `wp-content` is named and some websites rename it to something else and point to that newly named directory from inside `wp-config.php`.

To name a few heavily customized WordPress websites, I list only 2-3:
- [how to geek](https://www.howtogeek.com/)
- [Hongkiat](http://www.hongkiat.com/blog/)
- [Make tech easier](https://www.maketecheasier.com/)

Feel free to check their HTML code and see how they are doing things. There's no way it worths the effort to go so deep to reveal their back-end infrastructure since it's not that obvious.

In other words, it does not worth @AliasIO's time nor team's effort to add a feature that is strictly tailored on WordPress's needs.

If you **do** want to find what kind of plugin and or theme a website is using, I would suggest to use Chrome that contains extra information about a website via its own web developer tools that makes things clearer for you, as of which plugin(s) or theme is used.

By the way, it was an excellent question; kudos for asking it.

My $0.02 :+1:

**Comment by AliasIO** [comment]
I don't think there is a high enough demand for Wappalyzer plugins, interesting idea though.
